src/test/java/seedu/address/logic/parser/FindCommandParserTest.java

```diff
@@ -11,11 +11,14 @@
 
 public class FindCommandParserTest {
 
+    private static final String EMPTY_STRING = "     ";
```

Comment:
I think you can consider renaming this constant to SPACES instead of EMPTY_STRING as an empty string would tend to refer to "" in my opinion.